src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue17283.cs

```diff
@@ -0,0 +1,27 @@
+﻿#if ANDROID
+using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues
+{
+    internal class Issue17283 : _IssuesUITest
+	{
+		public override string Issue => "[Android] CarouselView doesn't scroll to the right Position after changing the ItemSource with Loop enabled";
+
+		public Issue17283(TestDevice testDevice) : base(testDevice) { }
+
+		[Test]
+		[Category(UITestCategories.CarouselView)]
+		public void Issue17283Test()
```

Comment:
Give the test a more significant name indicating what's being tested here. 
This is the name that shows up in the test results so it's easier if it has a valid description